packages/editor-ui/src/views/NodeView.vue

```diff
@@ -2746,12 +2737,8 @@ export default mixins(
 				await Promise.all(loadPromises);
 
 				if (this.defaultLocale !== 'en') {
-					try {
-						const headers = await this.restApi().getNodeTranslationHeaders();
-						addHeaders(headers, this.defaultLocale);
-					} catch (_) {
-						// no headers available
-					}
+					const headers = await this.restApi().getNodeTranslationHeaders();
+					if (headers) addHeaders(headers, this.defaultLocale);
```

Comment:
should not this be moved to App.vue? otherwise, won't apply if you load other pages

Reply:
I moved the headers up because they are used in credentials modal → `Details` → `Allow use by`. Otherwise they wouldn't be fetched if you look at this modal without first going through the canvas.

However, this surfaced an issue that the checkboxes after `Allow use by` are not getting shown if going directly from `/templates` to credentials modal and then to `Details`. I'm seeing this issue in English and non-English locales, so looks unrelated to i18n. Fix here or separately?

<img width="1043" alt="image" src="https://user-images.githubusercontent.com/44588767/161288424-eb74960b-8679-4eb4-8a6c-9cc06bd51ec5.png">
<img width="1019" alt="image" src="https://user-images.githubusercontent.com/44588767/161288498-aa6165b9-d95d-4dc3-934c-40e0c2a033cf.png">

Reply:
Templates bug so will be fixed separately.